skills/apple/test-modernizer/SKILL.md

```
@@ -0,0 +1,246 @@
+
+Remove `XCTestCase` inheritance. Prefer `struct` over `class`:
+
+- `final class FoodTruckTests: XCTestCase { ... }` -> `struct FoodTruckTests { ... }`
+
+### setUp/tearDown to init/deinit
+
+Replace `override func setUp()` with `init()` (can be `async throws`). Replace `override func tearDown()` with `deinit`. If `deinit` is needed, use
+`actor` or `final class` instead of `struct` (since structs have no `deinit`). Change stored properties to not use implicitly-unwrapped optional
+types, and move their initial assignment from `setUp` to either be initialized inline or, if the initialization is complex, in an initializer.
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)